error.go

```diff
@@ -221,3 +221,29 @@ var tooManyResultsError = &microerror.Error{
 func IsTooManyResults(err error) bool {
 	return microerror.Cause(err) == tooManyResultsError
 }
+
+var (
+	yamlConversionFailedErrorText = "error converting YAML to JSON:"
+)
+
+var yamlConversionFailedError = &microerror.Error{
+	Kind: "yamlConvertingFailedError",
+}
+
+// IsYamlConvertingFailed asserts yamlConvertingFailedError.
```

Comment:
```suggestion
// IsYamlConversionFailed asserts yamlConversionFailedError.
```

Reply:
I'm sorry, have a tough morning.

Reply:
OK no worries. Thanks for getting this sorted.